telescope/servicecontrol.h

```diff
@@ -0,0 +1,38 @@
+#ifndef SERVICECONTROL_H
+#define SERVICECONTROL_H
+
+#include <QObject>
+
+class ServiceControl : public QObject
+{
+    Q_OBJECT
+    Q_PROPERTY(QString serviceName READ serviceName WRITE setServiceName NOTIFY serviceNameChanged)
```

Comment:
I don't think we should expose that property. We probably only need to manage one service, so we can keep this encapsulated inside ServiceControl.